pages.nl/common/rg.md

```diff
@@ -2,36 +2,36 @@
 
 > Ripgrep is een recursieve regel-georiënteerde zoek tool.
```

Comment:
```suggestion
> Ripgrep, een recursieve regel-georiënteerde zoek tool.
```
This was changed in the english version, but I don't know dutch so I have no idea if this is better for your language.